Makefile

```diff
@@ -18,54 +18,75 @@
 ifeq ($(BOLOS_SDK),)
 $(error Environment variable BOLOS_SDK is not set)
 endif
-include $(BOLOS_SDK)/Makefile.defines
-
-#########
-#  App  #
-#########
 
-APP_LOAD_PARAMS = --curve ed25519
-APP_LOAD_PARAMS += --appFlags 0x200  # APPLICATION_FLAG_BOLOS_SETTINGS
-APP_LOAD_PARAMS += --path "44'/3030'"
-APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
+include $(BOLOS_SDK)/Makefile.defines
 
-APPVERSION_M = 1
-APPVERSION_N = 4
-APPVERSION_P = 2
-APPVERSION = $(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)
+########################################
+#        Mandatory configuration       #
+########################################
+# Application name
 APPNAME = Hedera
 
-COIN = HBAR
-
-DEFINES += $(DEFINES_LIB)
+# Application version
+APPVERSION_M = 1
+APPVERSION_N = 5
+APPVERSION_P = 0
+APPVERSION = "$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)"
 
+# Application source files
+APP_SOURCE_PATH  += src proto
+SDK_SOURCE_PATH  += lib_u2f
+
+# Application icons
+ICON_NANOS = icons/nanos_app_hedera.gif
+ICON_NANOX = icons/nanox_app_hedera.gif
+ICON_NANOSP = icons/nanox_app_hedera.gif
+ICON_STAX = icons/stax_app_hedera.gif
+ICON_FLEX = icons/flex_app_hedera.gif
+
+# Application allowed derivation curves.
+CURVE_APP_LOAD_PARAMS = ed25519
+
+# Application allowed derivation paths.
+PATH_APP_LOAD_PARAMS = "44'/3030'"   # purpose=coin(44) / coin_type=Hedera HBAR(3030)
+
+VARIANT_PARAM = COIN
+VARIANT_VALUES = hedera
```

Comment:
Previous definitiuon was `HBAR`... why a new one? What is the impact?

Reply:
I agree `HBAR `shall be kept

Reply:
Actually in the original makefile there was 

```
listvariants:
	@echo VARIANTS COIN hedera
```

and `COIN = HBAR`

I think `HBAR` is not used anywhere, but the `listvariants` is actually used by the SDK to compile the app (it is constructed in the standard makefile with `VARIANT_PARAM` and `VARIANT_VALUES`)

Reply:
Cf. C SDK : 

```
# Deployment scripts will use this Makefile target to retrieve the list of
# available variants and then call `make -j <VARIANT_PARAM>=<VALUE>` for each
# <VALUE> in <VARIANT_VALUES>.
listvariants:
	@echo VARIANTS $(VARIANT_PARAM) $(VARIANT_VALUES)
```

https://github.com/LedgerHQ/ledger-secure-sdk/blob/459c14bcdff748d1eac0204921cb06b90a73fa06/Makefile.standard_app#L264